python/src/aac/spec/spec.yaml

```diff
@@ -49,6 +49,16 @@ data:
         the specification of any plugins you may have installed.
 
         You can extend any 'enum' and 'data' definition.
+    - name: validation
+      type: Validation
+      description: |
+        A declaration for a validation definition.
+
+        Validation definitions must be paired with a corresponding validation
+        plugin. Validations are referenced by name in the definition that they
+        are applied to. For instance, the validation definition references a
+        validation plugin that ensures validation definitions have a corresponding
+        plugin capable of executing some validation check.
```

Comment:
Honestly, this "For instance"  part reads a bit strangely to me with references to validation definitions then plugins back to definitions then back to plugins. Perhaps we could reword a bit to specify more clearly why the validation definitions exist and what purpose they serve.